Makefile

```diff
@@ -207,6 +207,7 @@ qgc_firmware: \
 	check_mindpx-v2_default \
 	check_px4fmu-v1_default \
 	check_px4fmu-v2_default \
+	check_px4fmu-v2_lpe \
```

Comment:
Needs something here as well for the upload - https://github.com/PX4/Firmware/blob/master/.travis.yml#L82

I don't know how that line got so convoluted, but it needs to copy the firmware to s3deploy-branch/ renamed as px4fmu-v2_lpe.px4

If you expand the travis-ci output near the bottom you'll see it running.